docs/source/en/using-diffusers/other-formats.md

```diff
@@ -10,167 +10,218 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Load different Stable Diffusion formats
+# Model files and formats
 
 [[open-in-colab]]
 
-Stable Diffusion models are available in different formats depending on the framework they're trained and saved with, and where you download them from. Converting these formats for use in 🤗 Diffusers allows you to use all the features supported by the library, such as [using different schedulers](schedulers) for inference, [building your custom pipeline](write_own_pipeline), and a variety of techniques and methods for [optimizing inference speed](../optimization/opt_overview).
+Diffusion models are saved in various file types and organized in different layouts. Diffusers prefers storing model weights as `.safetensors` files in a multifolder layout, but it also supports loading file types (can be `.safetensors`, `.ckpt`, etc.) from a single file layout which is commonly used in the diffusion ecosystem.
```

Comment:
should we define the multifolder layout "diffusers-style format" and the other one "single-file format" and use these terms in the post? These are the terms already used by the community 

While we are trying to support loading the single-file format as first-class citizen, the multifolder layout is our official format, we prefer this format, and all our models are released in this format

Reply:
I think it may be better to reframe it as a "multifolder" and "single-file" layout instead of the current terms used by the community because:

1. It more accurately describes how the files are stored, avoiding confusion when discussing files and formats. For example, we talk about files being stored in the safetensors format, but when we talk about the "Diffusers-style" format, it may lead people to think we are talking about a file type rather than just the layout of how the files are organized.
2. It is more neutral because it doesn't position it as Diffusers vs single-file (community).

Reply:
Can we call it "diffuser-style multifolder"? I found it confusing without "diffusers" in the term:) 

It is a simple and plain fact that it is our format, and there is always a very clear difference from single-file. We will support single-file as best as we can, but the single-file checkpoints will always come from the community. And all the models released by us will be in this multifolder format.

Reply:
Updated to "Diffusers-multifolder" layout everywhere :)